chapters/ch10.md

```diff
@@ -18,7 +18,7 @@ Tiến hành một thí nghiệm cho tôi biết rõ rằng ý tưởng đó ho
 
 This is an iterative process. The faster you can go round this loop, the faster you will make progress. This is why having dev/test sets and a metric are important: Each time you try an idea, measuring your idea’s performance on the dev set lets you quickly decide if you’re heading in the right direction.
 ->
-Đây là một giai đoạn tốn rất nhiều thời gian. Nếu bạn đi qua cái vòng lặp này càng nhanh, bạn sẽ đạt được sự tiến triển càng nhanh.
+Đây là một giai đoạn có rất nhiều vòng lặp thử nghiệm. Nếu bạn đi qua cái vòng lặp này càng nhanh, bạn sẽ đạt được sự tiến triển càng nhanh. Đó là lý do tại sao có tập phát triển/thử nghiệm và một phép đo rất quan trọng: Mỗi khi bạn thử một ý tưởng mới, đo tính hiệu quả ý tưởng của bạn sẽ giúp bạn nhanh chóng quyết định rằng bạn có đang đi đúng hướng hay không.
```

Comment:
- This is an iterative process. --> Quy trình này sẽ lặp lại nhiều lần.

Reply:
done anh @ngcthuong